readme.md

```diff
@@ -1,16 +1,13 @@
-[<img alt="GitHub Workflow" src="https://img.shields.io/github/workflow/status/propensive/magnolia/Build/main?style=for-the-badge" height="24">](https://github.com/propensive/magnolia/actions)
-[<img src="https://img.shields.io/badge/gitter-discuss-f00762?style=for-the-badge" height="24">](https://gitter.im/propensive/magnolia)
-[<img src="https://img.shields.io/discord/633198088311537684?color=8899f7&label=DISCORD&style=for-the-badge" height="24">](https://discord.gg/CHCPjERybv)
-[<img src="https://img.shields.io/matrix/propensive.magnolia:matrix.org?label=MATRIX&color=0dbd8b&style=for-the-badge" height="24">](https://app.element.io/#/room/#propensive.magnolia:matrix.org)
-[<img src="https://img.shields.io/twitter/follow/propensive?color=%2300acee&label=TWITTER&style=for-the-badge" height="24">](https://twitter.com/propensive)
-[<img src="https://img.shields.io/maven-central/v/com.propensive/magnolia_2.12?color=2465cd&style=for-the-badge" height="24">](https://search.maven.org/artifact/com.propensive/magnolia_2.12)
-[<img src="https://vent.dev/badge/propensive/magnolia" height="24">](https://vent.dev/)
+[<img alt="GitHub Workflow" src="https://img.shields.io/github/workflow/status/softwaremill/magnolia/Build/main?style=for-the-badge" height="24">](https://github.com/softwaremill/magnolia/actions)
+[<img src="https://img.shields.io/maven-central/v/com.softwaremill/magnolia_2.12?color=2465cd&style=for-the-badge" height="24">](https://search.maven.org/artifact/com.softwaremill/magnolia_2.12)
```

Comment:
the group id for the badge should be `com.softwaremill.magnolia` I think

Reply:
👍